package.json

```diff
@@ -1,25 +1,6 @@
 {
     "private": true,
-    "scripts": {
-        "dev": "mix",
-        "development": "mix",
-        "watch": "mix watch",
-        "watch-poll": "mix watch -- --watch-options-poll=1000",
-        "hot": "mix watch --hot",
-        "prod": "mix --production",
-        "production": "mix --production"
-    },
-    "devDependencies": {
-        "cross-env": "^7.0.3",
-        "css-loader": "^5.2.6",
-        "laravel-mix": "^6.0.49",
-        "lodash": "^4.17.21",
-        "pace": "0.0.4",
-        "resolve-url-loader": "^4.0.0",
-        "sass": "^1.55.0",
-        "sass-loader": "^9.0.3",
-        "vue-template-compiler": "^2.7.8"
-    },
+    "scripts": { },
```

Comment:
Maybe we should delete this file entirely for now. Would make it the same as on the `v6` branch. We can add it afterwards if we decide to add `package.json` files to themes... or to the base theme... or to Backpack's core itself.